docs/usage/shoot/shoot-workload-identity.md

```diff
@@ -3,7 +3,7 @@ title: Shoot Workload Identity
 description: Configure access to infrastructure accounts via workload identity instead of static credentials
 ---
 
-# Shoot Workload Identity
+## Shoot Workload Identity
```

Comment:
Why the document starts with h2 and not h1?

Reply:
The MD linter throws 
> MD025/single-title/single-h1: Multiple top-level headings in the same documentmarkdownlint[MD025](https://github.com/DavidAnson/markdownlint/blob/v0.38.0/doc/md025.md)

Reply:
1. I am not sure if you are required to specify a `title` in the front-matter as it would be the same as it would be generated by the file name. IIRC, the convention was to specify title only when we have to overwrite the title. Here @Kostov6 and @n-boshnakov can share what is exactly the convention we follow and when we specify the title in the front-matter.
2. I am not sure from where this linter comes from. Shouldn't we ignore it as it is not imposed by a linter in Gardener and I am not aware of other files following the same?